samples/settings/boards/esp32.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_WIFI=y
+CONFIG_WIFI_ESP32=y
+CONFIG_HEAP_MEM_POOL_SIZE=131072
+
+CONFIG_NET_L2_ETHERNET=y
+
+CONFIG_NET_DHCPV4=y
+
+CONFIG_NET_CONFIG_LOG_LEVEL_DBG=y
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+
+CONFIG_MBEDTLS_ENTROPY_ENABLED=y
+CONFIG_MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED=y
+CONFIG_MBEDTLS_ECP_ALL_ENABLED=y
+
+
+CONFIG_ESP32_WIFI_STA_AUTO=y
+CONFIG_GOLIOTH_SAMPLE_WIFI=n
+
+CONFIG_MCUMGR=n
+CONFIG_MCUMGR_SMP_SHELL=n
+CONFIG_MCUMGR_CMD_CFG_MGMT=n
+CONFIG_MCUMGR_CMD_OS_MGMT=n
+
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_NVS=y
+
+CONFIG_HWINFO=y
```

Comment:
There is no rationale in commit message on why to enable HWINFO at this point, so we should remove it for now. If we want to use HWINFO for fetching hardware id from Zephyr shell, then we should do it for all boards **and** update README files to start using it.

Reply:
I'll remove from this PR this config because Alvaro will include it into another PR.

https://github.com/golioth/zephyr-sdk/pull/63